src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,37 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    static final int RANDOM_LIMIT = 100;
+
+    public Figure getRandomFigure() {
+        int indexOfFiguresType = (new Random().nextInt(5));
+        Figure result;
+        switch (indexOfFiguresType) {
+            case 1 :
+                result = new Triangle(new Random().nextInt(RANDOM_LIMIT),
```

Comment:
It's ok, but you can return the object immediately after creation

Reply:
done